src/constants.js

```diff
@@ -1,2 +1,3 @@
 export const API_URL =
-  'https://opentdb.com/api.php?amount=10&category=9&difficulty=easy&type=multiple';
+  'https://opentdb.com/api.php?amount=50&category=9&difficulty=easy&type=multiple';
```

Comment:
Extract common parts of the URLs like `https://opentdb.com/api.php` to avoid code duplication